cmd/config/config.go

```diff
@@ -124,7 +126,12 @@ var SubSystems = set.CreateStringSet([]string{
 	NotifyPostgresSubSys,
 	NotifyRedisSubSys,
 	NotifyWebhookSubSys,
-}...)
+)
+
+// SubSystemsDynamic - all sub-systems that have dynamic config.
+var SubSystemsDynamic = set.CreateStringSet(
+	CrawlerSubSys,
```

Comment:
We could add `HealSubSys` here? @klauspost - or should I add it in separate PR?

Reply:
I can add it. Just needed it in master before I could add it.

Reply:
@harshavardhana Added.